easygcm/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,9 @@
     <!-- GCM connects to Google Services. -->
     <uses-permission android:name="android.permission.INTERNET" />
 
+    <!-- delays gcm registration if device is offline -->
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```

Comment:
I think we should check if we have this permission also at runtime - the developer can force the manifest merger to exclude this permission (in case he doesn't want to have a new permission in the project). We would not be checking the connectivity change in case he doesn't have such permission.